proxy/envoy/testdata/envoy-v0.json.golden

```diff
@@ -1,7 +1,53 @@
 {
   "listeners": [
     {
-      "address": "tcp://0.0.0.0:1081",
+      "address": "tcp://0.0.0.0:80",
```

Comment:
It's difficult to annotate them. This PR is invasive which should not be usual. I think it's easier to make sense when the change is smaller.